static/app/components/projects/useProjectCreationAccess.tsx

```diff
@@ -4,8 +4,17 @@ import type {Organization} from 'sentry/types/organization';
  * Used to determine if viewer can see project creation button
  */
 export function useProjectCreationAccess({organization}: {organization: Organization}) {
```

Comment:
this is just a function, should rename since the linter forces rules of hooks with useXYZ. maybe hasProjectCreationAccess

Reply:
or maybe canCreateProject and just return a boolean